package.json

```diff
@@ -80,25 +83,27 @@
     "url": "^0.11.3",
     "util": "^0.12.5",
     "uuid": "^8.3.2",
+    "vitest": "^2.1.1",
     "web-vitals": "^0.2.4",
     "xlsx-preview": "^1.0.4"
   },
   "scripts": {
+    "postinstall": "patch-package",
     "preinstall": "node scripts/use-yarn.js",
     "prepare": "husky install",
     "dev": "craco start",
     "start": "craco start",
     "build": "craco build",
     "vercel:install": "yarn run add:npmrc && yarn install",
-    "test:unit": "jest src/ && jest test/unit",
+    "test:unit": "vitest run test/unit && vitest run src/ ",
```

Comment:
Interesting change, with Vitest the problems with global meta variable has gone?

Reply:
meta imports you mean? Yes.